src/cohortManager.js

```diff
@@ -0,0 +1,171 @@
+// COHORT MANAGER DATA SHAPE
+// cohorts[0].cohort = name1
+// cohorts[0].students.[0].firstName = a
+// const cohohorts = [
+//   {
+//     cohortName: 'Cohort 1',
+//     students: [
+//       {
+//         studentID: 1,
+//         firstname: 'a',
+//         lastname: 'b',
+//         gitusername: '@gitadesokan',
+//         email: 'ade@gmail.com'
+//       }
+//     ]
+//   },
+//   {
+//     cohortName: 'Chohort 2',
+//     students: [
+//       {
+//         studentID: 1,
+//         firstname: 'a',
+//         lastname: 'b',
+//         gitusername: '@gitadesokan',
+//         email: 'ade@gmail.com'
+//       }
+//     ]
+//   }
+// ]
+
+class CohortManager {
+  constructor() {
+    this.studentID = 0
+    this.cohorts = []
+  }
+
+  // QUESTION 1 => DOMAIN MODEL
+  // Create a cohort with a cohort name
+  // USER STORY =>
+  // - As an admin at boolean
+  // - So I can start a new batch of student
+  // - I'd like to create a new cohort
+
+  // DOMAIN MODEL
+  // - Methods => createNewCohort(cohortName)
+  // - Inputs => cohortName
+  // - Data => newCohort(@Object), Properties: cohortName(@String), students(@Array_Object)
+  // - Scenario => If admin enters a new cohort name
+  // - Outputs => newCohort(@object) is added to cohort manager
```

Comment:
fantastic use of commenting, i completely forgot about this and the fact that others will read my code xD